examples/sagemaker-tensorflow/README.md

```diff
@@ -0,0 +1,42 @@
+# Training and Serving Merlin on AWS SageMaker
+
+With AWS Sagemaker, you can package your own models that can then be trained
+and deployed in the SageMaker environment.
+The following notebook will show you how to use NVIDIA Merlin for training and
+inference in the SageMaker environment.
+
+- [Training and Serving Merlin on AWS SageMaker]
```

Comment:
does that supposed to have a link? if not can you remove the brackets?

Reply:
I meant to link it to the notebook. Thanks!